cuebot/src/main/resources/opencue.properties

```diff
@@ -96,3 +96,6 @@ history.archive_jobs_cutoff_hours=72
 
 # Delete down hosts automatically.
 maintenance.auto_delete_down_hosts=false
+
+# Set smtp host for mailing
```

Comment:
Based on this comment and the default value it's a little hard to understand what is supposed to be the value here. Maybe expand this to say something like "hostname/IP of the SMTP host"?

Reply:
Expanded it by saying "# Set hostname/IP of the smtp host. Will be used for mailing"